spruce/Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN apk add --update git \
   && go get -d github.com/geofffranks/spruce \
   && cd ${GOPATH}/src/github.com/geofffranks/spruce \
   && git checkout v${SPRUCE_VERSION} \
+  && export GOPATH=$GOPATH:$(pwd)/Godeps/_workspace \
```

Comment:
Minor thing: I'd put the Godeps workspace first in the GOPATH so that any packages there take precedence over ones in the existing GOPATH.